app/src/main/java/com/zulip/android/networking/AsyncFetchGoogleID.java

```diff
@@ -0,0 +1,18 @@
+package com.zulip.android.networking;
+
+import com.zulip.android.ZulipApp;
+
+public class AsyncFetchGoogleID extends ZulipAsyncPushTask {
+    /**
+     * Declares a new HumbugAsyncPushTask, passing the activity as context.
```

Comment:
Humbug is a deprecated name for the Zulip project; this comment can be updated to read "Declares a new *Zulip*AsyncPushTask ..."